### PR TITLE
Edits up to "THE READER"

### PR DESCRIPTION
**Pull request by bpringe**
Feel free to merge this and I may just keep using this branch for future edits. Great work on this!

### CODE REVIEW

**Comment by bpringe** [comment]
This PR also corrects the namespace to be `hello-clojure` instead of `hello_clojure`, this is consistent with mentions of the namespace in the guide using a hyphen.

File: drams/hello_clojure.clj [hunk 1]
```diff
@@ -47,24 +47,23 @@
...
   ;; (There are no statements.) Unless there is
-  ;; en error when evaluating the expressions there
+  ;; an error when evaluating an expression, there
   ;; is always a return value (which is sometimes `nil`).
 
   ;; An important aspect of this is that the result
   ;; of an expression is always the last form/expression
-  ;; evaluated. E.g. if you have a function defined
-  ;; like so:
+  ;; evaluated. Evaluate the function below:
```
Comment (PEZ) [inline drams/hello_clojure.clj]:
```suggestion
  ;; evaluated. E.g. if you have a function defined
  ;; like so:
```

File: drams/hello_clojure.clj [hunk 2]
```diff
@@ -121,7 +121,7 @@
   map             ; symbol
   +               ; symbol - most punctuation allowed
   clojure.core/+  ; namespaced symbol
-  nil             ; null value
+  nil             ; nil value
```
Comment (PEZ) [inline drams/hello_clojure.clj]: That is debatable. 😄 
https://clojure.org/guides/learn/syntax#_symbols_and_idents

```suggestion
  nil             ; null/nil value (named in the LISP tradition)
```

File: drams/hello_clojure.clj [hunk 3]
```diff
@@ -315,8 +316,8 @@ like this, if leading spaces are no-no."
   ;; the special forms section up with just noting that
   ;; together with _how_ Clojure reads and evaluates code,
   ;; the special forms make up the Clojure language
-  ;; itself. The next level och building blocks are
-  ;; macros. But let's investigate this with how code
+  ;; itself. The next level of building blocks are
+  ;; macros. But let's investigate how code
```
Comment (PEZ) [inline drams/hello_clojure.clj]:
```suggestion
 ;; itself. The next level och building blocks are macros.
 ;;.But let us investigate this thing with how code```

**Comment by PEZ** [comment]
Awesome. Thanks!

**Pull request merged**